pkg/verifier/notation/truststore.go

```diff
@@ -32,37 +32,40 @@ var logOpt = logger.Option{
 }
 
 type trustStore struct {
-	certPaths  []string
-	certStores map[string][]string
+	certPaths        []string
+	certStoresByType map[string]interface{}
```

Comment:
since we'll do the conversion in notation.go, the `certStoresByType` is always map[string]map[string][]string, please correct me if I'm wrong.